packages/neos-ui-guest-frame/src/InlineUI/index.js

```diff
@@ -48,10 +48,8 @@ export default class InlineUI extends PureComponent {
         const canBeDeleted = $get('policy.canRemove', this.props.focusedNode) || false;
         const canBeEdited = $get('policy.canEdit', this.props.focusedNode) || false;
         const visibilityCanBeToggled = !$contains('_hidden', 'policy.disallowedProperties', this.props.focusedNode);
-        // Check if focusedNode is in Guest frame to prevent errors (e.g. if still in store)
-        const focusedNodeInGuestFrame = findNodeInGuestFrame(focusedNodeContextPath) ? focused : {};
 
-        return (
+        return (findNodeInGuestFrame(focusedNodeContextPath) &&
```

Comment:
Hey, sorry for being picky, but isn't it enough to just check for presence of `focused`? Why do we need to find it in the guest frame?

Reply:
Being picky is the way to a good product, so keep challenging me ;) The focussed element is reset in another saga as the change of the iframe src which can lead to the new content rendering without the focussed node being present thus leading to errors. This is also checked in the toolbar itself now so we could skip the check here but would end up with an empty container in the dom, since the toolbar returns null.

Reply:
Removed that as well. I tried to figure out what the inlineUI-Component is for in difference to the Toolbar. I thought i figured it out seeing the Toolbar as a presentational component (only containing UI state) and the InlineUI-Component as a container with connection to the redux store...But if I'm right with that, shouldn't the decision, if the Toolbar is rendered, be made in the container-Component getting the relevant nodeElement there as well? Or did I find a pattern, that wasn't really intended?